# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1397402838 14400
#      Sun Apr 13 11:27:18 2014 -0400
# Node ID 93d0fc8a5d868c4c34a2c68177ec8d1df06add45
# Parent  ce36233e4849de89ef1a460bd74bba44abf2088c
build: Add missing octave-default-image.h to source distribution

* libinterp/corefcn/module.mk (COREFCN_INC): Add octave-default-image.h
file introduced in cset 2deed6538c72.

diff --git a/libinterp/corefcn/module.mk b/libinterp/corefcn/module.mk
--- a/libinterp/corefcn/module.mk
+++ b/libinterp/corefcn/module.mk
@@ -86,16 +86,17 @@ COREFCN_INC = \
   corefcn/oct-map.h \
   corefcn/oct-obj.h \
   corefcn/oct-prcstrm.h \
   corefcn/oct-procbuf.h \
   corefcn/oct-stdstrm.h \
   corefcn/oct-stream.h \
   corefcn/oct-strstrm.h \
   corefcn/oct.h \
+  corefcn/octave-default-image.h \
   corefcn/octave-link.h \
   corefcn/pager.h \
   corefcn/pr-output.h \
   corefcn/procstream.h \
   corefcn/profiler.h \
   corefcn/sighandlers.h \
   corefcn/siglist.h \
   corefcn/sparse-xdiv.h \
